feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649f12c7c84ffb02b0fc7563.md

---
id: 649f12c7c84ffb02b0fc7563
title: Step 17
challengeType: 0
dashedName: step-17
---

# --description--

> Write an `else` statement inside which `rolls` will be incremented by `1`. After that, call the  `rollDice` function.

Reviewer: ```suggestion
Write an `else` statement. In the statement, use the increment operator to increment `rolls` by one. Then call the `rollDice` function.
```

If we are not allowing syntax like `rolls+=1` or `rolls=rolls+1`, we should specify using the increment operator.

> Write an `else` statement where `rolls` will be incremented by `1` and you will call the function named `rollDice` here.

Reviewer 2: ```suggestion
Write an `else` statement inside which `rolls` will be incremented by `1`. After that, call the  `rollDice` function.
```

# --hints--

> You should add an `else` statement where `rolls` will be incremented by `1` 

Reviewer: Split into two tests.

```js
assert.match(code, /else\s*{\s*rolls\s*\+\+\s*[^}]*}/);
```

> You should call the function named `rollDice` here.

Reviewer: ```suggestion
After incrementing `rolls`, call the `rollDice` function.
```

Where is "here"? Too vague.
